OpenOversight/app/main/views.py

```diff
@@ -312,8 +310,10 @@ def redirect_officer_profile(officer_id: int):
 def officer_profile(officer_id: int):
     form = AssignmentForm()
     try:
-        officer = Officer.query.filter_by(id=officer_id).one()
-        officer.incidents.query.order_by(Incident.date.desc(), Incident.time.desc())
+        officer = db.session.get(Officer, officer_id)
+        officer.incidents = sorted(
+            officer.incidents, key=lambda i: (i.date, i.time), reverse=True
+        )
```

Comment:
Validate on the below test:
<img width="696" alt="Screenshot 2024-07-21 at 10 15 44 PM" src="https://github.com/user-attachments/assets/aaa0bd45-534b-43ae-9e7b-90d15b6c2ee6">